packages/loot-core/src/server/accounts/sync.js

```diff
@@ -781,11 +784,11 @@ export async function syncNordigenAccount(userId, userKey, id, acctId, bankId) {
       date,
     );
 
-    if (transactions.length === 0) {
+    if (transactions.all.length === 0) {
```

Comment:
If we apply the suggestion in comment above - we don't need to manually pick `.all` everywhere. Thus the code is a bit cleaner :)